tests/geoLocationCloud.test.js

```diff
@@ -107,19 +107,19 @@ if (isAsync) {
     await testAvailableProperties(done, pipeline, engine);
   });
 
-  test('Available Properties Digital Element', async done => {
-    if (myResourceKey === '!!YOUR_RESOURCE_KEY!!') {
-      throw new Error('No resource key is present!');
-    }
-
-    const pipeline = new GeoLocation.GeoLocationPipelineBuilder({
-      resourceKey: myResourceKey,
-      locationProvider: 'digitalelement'
-    }).build();
-    var engine = pipeline.flowElements.location_digitalelement;
-
-    await testAvailableProperties(done, pipeline, engine);
-  });
+  // test('Available Properties Digital Element', async done => {
```

Comment:
The two tests commented out here can actually be removed completely now, as that provider is no longer supported.